build/Nuget/Microsoft.NET.Sdk.Nuget.targets

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <Target Name="LayoutSdkPackage">
+  <Target Name="LayoutSdkPackage" BeforeTargets="Build" >
```

Comment:
Diff would need clearer if build still depended on it. Switching to beforetargets is unrelated to the task at hand.